ada_feeding/scripts/ada_planning_scene.py

```diff
@@ -34,6 +34,7 @@
 from tf2_ros.buffer import Buffer
 from tf2_ros.static_transform_broadcaster import StaticTransformBroadcaster
 from tf2_ros.transform_listener import TransformListener
+from pyquaternion import Quaternion as Q
```

Comment:
Don't use this. It seems like the only place you use this is lines 694-708. See below comment for suggestions on what to do instead.